addons/mass_mailing/static/src/snippets/s_rating/001.scss

```diff
@@ -8,12 +8,17 @@
         display: flex;
         align-items: center;
 
-        .s_rating_title {
+        .s_rating_title, > :first-child {
```

Comment:
Add a comment to explain that `> :first-child` is there for compatiblity with the old version of the snippet where the class didn't yet exist.